shell/edit/provisioning.cattle.io.cluster/index.vue

```diff
@@ -449,10 +450,14 @@ export default {
       }
     },
 
+    filteredSubTypes() {
+      return this.subTypes.filter((subtype) => !subtype.hidden);
+    },
+
     groupedSubTypes() {
       const out = {};
 
-      for ( const row of this.subTypes ) {
+      for ( const row of this.filteredSubTypes ) {
```

Comment:
When all hosted / cluster providers are disabled there's a nit picky vertical space issue

![image](https://github.com/user-attachments/assets/2baa8908-db62-4863-8fa1-a9a62b93b801)

The gap between line and `Provision new nodes and...` isn't there and isn't the same as above and below `Use existing nodes`. It's not been introduced in this PR but will be more noticeable without it

Reply:
updated to make the spacing consistent across all 3 sections:
![Screenshot 2024-12-19 at 11 57 26 AM](https://github.com/user-attachments/assets/5233aac2-77ee-4220-b884-810547d8a01e)
![Screenshot 2024-12-19 at 11 57 10 AM](https://github.com/user-attachments/assets/c1ef3a71-1cc9-4de4-bb2a-99e8a7d1c44a)